docs/source/guide/index.md

```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
 
 ## Label Studio terminology
 
-When you upload data to Label Studio, each item in the dataset becomes a labeling task. The following table describes some terms you might encounter as you use Label Studio.
-
-| Term | Description |
-| --- | --- |
-| Dataset | What you import into Label Studio, comprised of individual items, or labeling tasks. |
-| Task | A distinct item from a dataset that is ready to be labeled, pre-annotated, or has already been annotated. For example: a sentence of text, an image, or a video clip. |
-| Region | The portion of the task identified for labeling. For images, an example region is a bounding box. For text, an example region is a span of text. Often has a label assigned to it. | 
-| Labels | What you add to each region while labeling a task in Label Studio. |
-| Relation | A defined relationship between two labeled regions. |
-| Result | A label applied to a specific region as stored in an annotation or prediction. See [Label Studio JSON format of annotated tasks](export.html#Label-Studio-JSON-format-of-annotated-tasks). |
-| Predictions | What machine learning models create for an unlabeled dataset. |
-| Pre-annotations | Predicted annotations in Label Studio format, either in a file or from a machine learning backend. See [import pre-annotations](predictions.html).
-| Annotations | The output of a labeling task. Previously called "completions". |
-| Templates | Example labeling configurations that you can use to specify the type of labeling that you're performing with your dataset. See [all available templates](/templates) |
-| Tags | Configuration options to customize the labeling interface. See [more about tags](/tags). |
+When you upload data to Label Studio, each item in the dataset becomes a labeling task. The following table defines the terms that appear throughout Label Studio.
+
+**Annotation**
+Annotation is the process of labeling data in various formats such as video, images, or text, so that machines can understand it. The output of a labeling task which was previously called **Completions**. 
+
+**Artificial Intelligence (AI)**
+AI is the intelligence demonstrated by machines, as opposed to the natural intelligence displayed by animals including humans.  It refers to the simulation of human intelligence in machines that are programmed to think like humans and mimic their actions.
+
+**Audio classification**
+Audio classification or sound classification can be referred to as the process of analyzing audio recordings. This amazing technique has multiple applications in the field of AI and data science such as chatbots, automated voice translators, virtual assistants, music genre identification, and text to speech applications.
+
+**Audio labeling**
```

Comment:
[Audio labeling](/audio_regions.html#Labeling-Configuration) or speech labeling is a standard annotation technique that concerns separating sounds and labeling with specific metadata.
\